5/windows/windowsservercore/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# escape=`
+FROM microsoft/windowsservercore
```

Comment:
Once https://github.com/docker-library/openjdk/pull/88 is merged and Windows images are available it should depend on these ones.